src/services/commentService.ts

```diff
@@ -0,0 +1,7 @@
+import suggestionService from "./suggestionService";
+export default {
+  add: (comments: object[], comment: object, id: number, target: object) => {
```

Comment:
Tipar com interface e não object
```suggestion
import suggestionService from "./suggestionService";

export default {
  add: (comments: object[], comment: object, id: number, target: object) => {
```